sdk/state/open.go

```diff
@@ -8,8 +8,7 @@ import (
 	"github.com/stellar/go/xdr"
 )
 
-// TODO - should store on channel like Update and Close proposals?
-type Open struct {
+type OpenAgreement struct {
```

Comment:
🎉 +1 this is a great change. We can now say that "Agreements" are a concept. I think this is a concept we should name in the SEP too.